database/changesets/20201101_CommonTable.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="jwarren@broadinstitute.org"
+               dbms="postgresql"
+               id="20201101"
+               logicalFilePath="20201101_CommonTable.xml">
+        <comment>
+            A table containing content common within workflows in a workload.
```

Comment:
Am I correct in that you propose to only have one of these tables and all workloads *could* write into this table?
It's a nice idea, but I wonder if this will lead to greater database contention - hence why we're using separate tables for each workload.

Maybe you can discuss this in the meeting later.

Reply:
This table is a is a sorta like 1-to-0..1 mapping from the workload table to this one. Each row in the workload table can have either 0 or 1 rows in this one. The "interface" I've written over this table in the `common.clj` file (poorly named, couldn't think of something better) treats these rows as immutable

Reply:
So exactly as I had said above. Did you decide in the meeting that this wasn't an issue?